website/docusaurus.config.js

```diff
@@ -67,13 +67,17 @@ module.exports = {
   onBrokenLinks: 'ignore',
   onBrokenMarkdownLinks: 'ignore',
   themeConfig: {
-    autoCollapseSidebarCategories: true,
+    docs: {
+      sidebar: {
+        autoCollapseCategories: true,
+        },
+    },  
     prism: {
       additionalLanguages: ['rust', 'swift'],
     },
     algolia: {
-      appId: process.env.DOCSEARCH_APP_ID,
-      apiKey: process.env.DOCSEARCH_API_KEY,
+      appId: process.env.DOCSEARCH_APP_ID ||  "local",
+      apiKey: process.env.DOCSEARCH_API_KEY || "local",
       indexName: 'suborbital',
       contextualSearch: true
     },
```

Comment:
Would it be better to conditionally include this stanza only when `process.env.DOCSEARCH_{APP_ID,API_KEY}` are set, rather than stubbing in fake values?

Reply:
I think the fake values are there to handle the case when ```process.env.DOCSEARCH_{APP_ID,API_KEY}``` are *not* set (which only happens when the docs are running locally), because the config will look for them. They're set in a workflow that isn't triggered when the docs run locally, so without the dummy values running locally fails.

Reply:
Okay, so @arbourd says @callahad is suggesting a different method. I really need to merge this so I can run the docs locally, though, so I'm just going to do that and put this on my to-do list to revisit and optimize later. :)